Core/src/IfcGeometryConverter/SplineConverter.h

```diff
@@ -198,9 +197,7 @@ namespace OpenInfraPlatform
 						double step;
 						std::tie(knotStart, knotEnd, step) = obtainKnotRange(order, knotArray, numCurvePoints);
 
-						std::vector<double> length;
-						length.reserve(numCurvePoints);
-						std::vector<double> curvature;
+						std::vector<std::pair<double, double>> curvature;

```

Comment:
As the naming goes, `lengthsWithCurvatures` would be more descriptive, as there are not only curvatures in the vector.

Reply:
Done in commit b7aa29d.